apps/api/e2e/tests/rewards-api.test.ts

```diff
@@ -178,4 +182,86 @@ describe("Rewards API", () => {
       });
     });
   });
+
+  test("getRewardsWithProofs returns empty when allocator returns null transaction hash", async () => {
```

Comment:
this is test uses a mock allocator that returns null for tx ([see](https://github.com/recallnet/js-recall/pull/1575/files#diff-c72efd1ee9c40f10a0405222df6902cdbc2c75ae3818c1d406799c13c62bba4cR220)), and we can see that `getRewardsWithProofs` returns an empty response